.github/workflows/pr-checks.yml

```diff
@@ -0,0 +1,19 @@
+name: PR Checks
+
+on: pull_request
+
+jobs:
+  pr-checks:
+    runs-on: macos-latest
+
+    steps:
+      - name: Clone the Repository
+        uses: actions/checkout@v3
+
+      - name: Install Deno
+        uses: denoland/setup-deno@v1
+        with:
+          deno-version: v1.x
```

Comment:
Using latest Deno. We might consider pinning a specific version number in the future.